src/Microsoft.ML.Data/Data/Conversion.cs

```diff
@@ -53,9 +53,12 @@ internal sealed class Conversions
 
         // REVIEW: Reconcile implementations with TypeUtils, and clarify the distinction.
 
-        // Singleton pattern.
+        // Options for the DoubleParser, currently the TextLoader would use non-default options
+        private DoubleParser.Options _doubleParserOptions;
+
+        // Default instance used by most of the codebase
         private static volatile Conversions _instance;
-        public static Conversions Instance
+        public static Conversions Instance // MYTODO: Should I rename this to "DefaultInstance"? It is used in over 99 places on the codebase
```

Comment:
Should I rename this? I guess it's not necessary. Although in ValueCreatorCache I did rename "Instance" to "DefaultInstance" in order to suggest that multiple instances could exist now.

Reply:
Yeah, that is a good idea. The `.Instance` convention is usually to indicate singletons. 

---
In reply to: [429523634](https://github.com/dotnet/machinelearning/pull/5154#discussion_r429523634) [](ancestors = 429523634)